bedrock-devnet/devnet/__init__.py

```diff
@@ -143,7 +143,7 @@ def devnet_l1_allocs(paths):
 
     fqn = 'scripts/Deploy.s.sol:Deploy'
     run_command([
-        'forge', 'script', fqn, "--sig", "runWithStateDump()"
+        'forge', 'script', fqn, "--sig", "runWithStateDump()", "--sender", "0x90F79bf6EB2c4f870365E785982E1f101E93b906"
```

Comment:
We need to set the sender here to an account we know the private key of, because the sender ends up being the owner of the ProxyAdmin SAFE (which we need to enable the CGT feature).

Reply:
Oh i see, apologies for this, could you include a comment here saying that?

Reply:
[94f901a](https://github.com/ethereum-optimism/optimism/pull/10440/commits/94f901a44406e860f867a173f9004054d237d638)